### PR TITLE
Tests Time Refactoring 

### PR DESCRIPTION
**Pull request by ralikio**
**Description**

Changes proposed in this pull request:

- Tests Refactoring to decrease overall time needed to run them.
- Removed unnecessary sleeps in schema migrator.

**Related issue(s)**

### CODE REVIEW

File: components/kyma-environment-broker/internal/broker/client.go [hunk 1]
```diff
@@ -10,8 +10,8 @@ import (
 	"time"
 
 	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+	"github.com/pkg/errors"
```
Comment (wozniakjan) [inline components/kyma-environment-broker/internal/broker/client.go:13]: please don't use this archived package https://github.com/kyma-project/control-plane/pull/2234

File: components/kyma-environment-broker/internal/broker/poller.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
+package broker
+
+import (
+	"time"
+
+	"github.com/pkg/errors"
```
Comment (wozniakjan) [inline components/kyma-environment-broker/internal/broker/poller.go:6]: same here, please avoid usage of this deprecated package https://github.com/kyma-project/control-plane/pull/2234

File: components/kyma-environment-broker/cmd/broker/broker_suite_test.go [hunk 3]
```diff
@@ -460,15 +491,15 @@ func (s *BrokerSuiteTest) FailDeprovisioningOperationByProvisioner(operationID s
 
 func (s *BrokerSuiteTest) FinishDeprovisioningOperationByProvisioner(operationID string) {
 	var op *internal.DeprovisioningOperation
-	err := wait.PollImmediate(pollingInterval, 2*time.Second, func() (done bool, err error) {
+	err := s.poller.Invoke(func() (err error) {
 		op, err = s.db.Operations().GetDeprovisioningOperationByID(operationID)
 		if err != nil {
-			return false, nil
+			return err
```
Comment (ralikio) [inline components/kyma-environment-broker/cmd/broker/broker_suite_test.go:497]: This corrects previous problem of hidden error.

**Review by wozniakjan — APPROVED** [review]

**Review by piotrmiskiewicz — APPROVED** [review]

**Review by piotrmiskiewicz — APPROVED** [review]

**Pull request merged**